docs/development/mcp-tooling.md

```
@@ -94,6 +94,73 @@ subcommands and any option token after `run`, including `--agent`, `--attach`,
+`--strict-mcp-config`, `--setting-sources ""`, `--tools`, and ephemeral
+skill / settings / MCP injection). The OpenCode harness above remains a fully
+supported peer; both are documented side-by-side and there is no deprecation:
+
+```bash
+agent-orchestrator claude
+agent-orchestrator-claude
+```
+
+The Claude launcher builds a per-launch ephemeral envelope under the system
+temp directory and spawns `claude` with `cwd = <envelope>`. The envelope
+contains a curated `.claude/skills/` (orchestrate-* only), a deny-by-default
```

markdownlint-cli2 (0.22.1)

[warning] Spaces inside emphasis markers

(MD037, no-space-in-emphasis)